app/views/downtimes/_form.html.erb

```diff
@@ -7,32 +7,32 @@
   year: {
     name: "downtime[start_time(1i)]",
     label: "Year",
-    value: params.dig("downtime", "start_time(1i)"),
+    value: @downtime.start_time&.year || params.dig("downtime", "start_time(1i)"),
```

Comment:
The changes in here to set the value (I think it's these changes) seems to have had the side-effect of meaning that when a value is omitted from one of the text boxes it gets the value "1" put into it when the validation errors are shown.

(We should probably add some test coverage around this.)